docs/_posts/2022-05-06-writing-cli-tools-with-typescript.md

```diff
@@ -0,0 +1,187 @@
+# Writing CLI tools with TypeScript
+
+## Introduction
```

Comment:
You shouldn't start with a sub head line but instead with a short introduction which will also be used as excerpt.

Reply:
![image](https://user-images.githubusercontent.com/2569015/166449709-be43db0c-da72-4485-a9f4-35aecdba924f.png)